atlas_densities/app/data/mtypes/probability_map/probability_map.csv

```diff
@@ -1,19 +1,17 @@
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.06250000000000001,0.0,0.0,0.05208333333333331,0.8854166666666667,0.0,0.0,0.0,L2/3 Sst-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.8452380952380955,0.002976190476190401,0.15178571428571422,0.0,0.0,0.0,L2/3 Vip-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.011494252873563201,0.0,0.0,0.25287356321839083,0.7011494252873565,0.011494252873563201,0.0,0.022988505747126402,L4 Htr3a-Cre_NO152
-0.0,0.0,0.0,0.0,0.0,0.0,0.005000000000000003,0.012298850574712606,0.05290697674418602,0.11662790697674416,0.34650494520181785,0.3047647687784016,0.14229885057471264,0.0,0.01959770114942521,L4 Pvalb-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0125,0.025,0.0625,0.0125,0.4375,0.0625,0.35,0.0,0.0375,L4 Sst-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,0.0,L4 Vip-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.005813953488372002,0.010438385458433503,0.011317829457364304,0.42674418604651176,0.18421812349639136,0.15185779203421546,0.17803528468323984,0.006666666666666603,0.024907778668805107,L5 Htr3a-Cre_NO152
-0.0,0.0,0.0,0.0,0.0,0.0,0.0019230769230769006,0.008653846153846104,0.035256410256410214,0.0019230769230769006,0.3661858974358975,0.07772435897435893,0.42564102564102585,0.025641025641025605,0.05705128205128202,L5 Pvalb-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.009827147494954802,0.0032715376226826005,0.020408163265306103,0.22129928570533242,0.5455924670473613,0.16686095338372253,0.006543075245365301,0.026197370235274905,L5 Sst-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.011494252873563201,0.0,0.0,0.25287356321839083,0.7011494252873565,0.011494252873563201,0.0,0.022988505747126402,L6a Htr3a-Cre_NO152
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.033333333333333305,0.0,0.3833333333333334,0.0,0.4833333333333334,0.033333333333333305,0.06666666666666661,L6a Pvalb-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.010178117048346005,0.016200169635284107,0.0,0.35169635284139117,0.13231552162849874,0.41556403731976266,0.02128922815945711,0.05275657336726032,L6a Sst-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.015267175572519004,0.007633587786259502,0.0,0.33587786259541985,0.19847328244274814,0.38167938931297724,0.015267175572519004,0.04580152671755721,L6a Vip-IRES-Cre
-0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.011494252873563201,0.0,0.0,0.25287356321839083,0.7011494252873565,0.011494252873563201,0.0,0.022988505747126402,L6b Htr3a-Cre_NO152
+,BP_bAC,BP_bIR,BP_bNAC,BP_cAC,BP_cNAC,BP_dSTUT,BTC_bAC,BTC_bIR,BTC_bNAC,BTC_cAC,BTC_cNAC,ChC_cAC,ChC_cNAC,ChC_dNAC,DBC_bAC,DBC_bIR,DBC_bNAC,DBC_cAC,LBC_bAC,LBC_bNAC,LBC_cAC,LBC_cNAC,LBC_cSTUT,LBC_dNAC,MC_bAC,MC_bNAC,MC_cAC,MC_cNAC,MC_dNAC,NBC_bAC,NBC_bNAC,NBC_cAC,NBC_cIR,NBC_cNAC,NBC_dNAC,NGC_bNAC,NGC_cAC,NGC_cNAC,NGC_cSTUT,SBC_bNAC,SBC_cAC,SBC_dNAC,BTC_bSTUT,BTC_dNAC,DBC_bSTUT,DBC_cIR,DBC_cNAC,LBC_dSTUT,LBC_cIR,MC_bIR,MC_bSTUT,MC_cSTUT,NBC_bIR,NBC_bSTUT,NBC_cSTUT,NBC_dSTUT,LBC_bIR,LBC_bSTUT,MC_cIR,DAC_bNAC,DAC_cNAC,DLAC_cNAC,HAC_bNAC,HAC_cIR,HAC_cNAC,NGC-DA_bNAC,NGC-DA_cAC,NGC-DA_cNAC,NGC-DA_cSTUT,NGC-SA_cNAC,SLAC_bNAC,SLAC_cAC,SLAC_cNAC
```

Comment:
can we keep the `molecular_type` heading here; and arrange the CSV file the same way, so it's clear what has changed?

Reply:
I do not think it makes sense that the last column to be the heading of each row, but I will put back the  `molecular_type` label.